### PR TITLE
DOC: Tidy Annotation Docstrings

### PR DESCRIPTION
**Pull request by John-P**
- Make line wrapping consistent
- Make indentation consistent
- Fix typos

### CODE REVIEW

File: tiatoolbox/annotation/dsl.py [hunk 1]
```diff
@@ -20,41 +20,41 @@
...
-should be faster than post-query filtering in python or filtering
-during the query via a registered custom function callback.
-
-An additional benefit is that the same input string can be
-used across different backends. For example, the previous
-simple example predicate string can be evaluated as both a valid
-python expression and can be converted to an equivalent valid SQL
-expression simply by running `eval` with a different set of globals
-from this module.
+This modules facilitates conversion from a restricted subset of python
```
Comment (shaneahmed) [inline tiatoolbox/annotation/dsl.py:23]:
```suggestion
This module facilitates conversion from a restricted subset of python
```

File: tiatoolbox/annotation/storage.py [hunk 2]
```diff
@@ -188,7 +189,8 @@ def _geometry_predicate(name: str, a: Geometry, b: Geometry) -> Callable:
                 The second geometry.
 
         Returns:
-            bool: True if the predicate holds.
```
Comment (shaneahmed) [inline tiatoolbox/annotation/storage.py:191]: line 174 `geometric`

File: tiatoolbox/annotation/storage.py [hunk 3]
```diff
@@ -188,7 +189,8 @@ def _geometry_predicate(name: str, a: Geometry, b: Geometry) -> Callable:
                 The second geometry.
```
Comment (shaneahmed) [inline tiatoolbox/annotation/storage.py:189]: line 188, `Geometry`

File: tiatoolbox/annotation/storage.py [hunk 4]
```diff
@@ -346,22 +350,20 @@ def patch(
...
-        annotation use `__setitem__`.
+        Partial update of an annotation. Providing only a geometry will update
+        the geometry and leave properties unchanged. Providing a properties
+        dictionary applies a patch operation to the properties. Only updating
+        the properties which are given and leaving the rest unchanged. To
+        completely replace an annotation use `__setitem__`.
 
         Args:
             key(str):
                 The key of the annoation to update.
```
Comment (shaneahmed) [inline tiatoolbox/annotation/storage.py:361]:
```suggestion
                The key of the annotation to update.
```

File: tiatoolbox/annotation/storage.py [hunk 5]
```diff
@@ -346,22 +350,20 @@ def patch(
...
             geometry(Geometry):
                 The new geometry. If None, the geometry is not updated.
             properties(dict):
-                A dictionary of properties to patch and their new
-                new values. If None, the existing properties are not
-                altered.
+                A dictionary of properties to patch and their new new values.
```
Comment (shaneahmed) [inline tiatoolbox/annotation/storage.py:365]:
```suggestion
                A dictionary of properties to patch and their new values.
```

File: tiatoolbox/annotation/storage.py [hunk 6]
```diff
@@ -532,47 +538,41 @@ def query(
...
-                used across different backends (e.g. the previous
-                example predicate string is valid for both
-                `DictionaryStore `and a `SQliteStore`). On the
-                other hand it has many more limitations. It is important
-                to note that untrusted user input should never be
-                accepted to this argument as arbitrary code can be run
-                via pickle or the parsing of the string statement.
+                A statement which should evaluate to a boolean value. Only
+                annotations for which this predicate is true will be returned.
+                Defaults to None (assume always true). May be a string,
```
Comment (shaneahmed) [inline tiatoolbox/annotation/storage.py:546]:
```suggestion
                Defaults to None (assume always true). Maybe a string,
```

File: tiatoolbox/annotation/storage.py [hunk 7]
```diff
@@ -532,47 +538,41 @@ def query(
...
+                callable, or pickled function as bytes. Callables are called to
+                filter each result returned the from annotation store backend
```
Comment (shaneahmed) [inline tiatoolbox/annotation/storage.py:548]:
```suggestion
                filter each result returned from the annotation store backend
```

File: tiatoolbox/annotation/storage.py [hunk 8]
```diff
@@ -613,46 +613,40 @@ def iquery(
...
-                during the query. Additionally, the same string can be
-                used across different backends (e.g. the previous
-                example predicate string is valid for both
-                `DictionaryStore `and a `SQliteStore`). On the
-                other hand it has many more limitations. It is important
-                to note that untrusted user input should never be
-                accepted to this argument as arbitrary code can be run
-                via pickle or the parsing of the string statement.
+                Only annotations for which this predicate is true will be
+                returned. Defaults to None (assume always true). May be a
```
Comment (shaneahmed) [inline tiatoolbox/annotation/storage.py:620]:
```suggestion
                returned. Defaults to None (assume always true). Maybe a
```

File: tiatoolbox/annotation/storage.py [hunk 9]
```diff
@@ -613,46 +613,40 @@ def iquery(
...
+                string, callable, or pickled function as bytes. Callables are
+                called to filter each result returned the from annotation store
```
Comment (shaneahmed) [inline tiatoolbox/annotation/storage.py:622]:
```suggestion
                called to filter each result returned from the annotation store
```

File: tiatoolbox/annotation/storage.py [hunk 10]
```diff
@@ -1307,7 +1312,8 @@ def _query(
                 during the query.
 
         Returns:
-            sqlite3.Cursor: A database cursor for the current query.
```
Comment (shaneahmed) [inline tiatoolbox/annotation/storage.py:1310]: line 1308, `comparing`

File: tiatoolbox/annotation/dsl.py [hunk 11]
```diff
@@ -349,10 +355,13 @@ def sql_list_sum(x: SQLJSONDictionary) -> SQLTriplet:
     """Return a representation of the sum of a list.
 
     Args:
-        x: The list to sum.
+        x:
```
Comment (shaneahmed) [inline tiatoolbox/annotation/dsl.py:358]: type?

**Review by shaneahmed — COMMENTED** [review]
Thanks @John-P 
Please see my comments attached.

File: tiatoolbox/annotation/storage.py (hunk 10 above)
Comment (John-P) [reply tiatoolbox/annotation/storage.py:1310]: Sorry, but I don't understand this comment

Comment (shaneahmed) [reply tiatoolbox/annotation/storage.py:1310]: https://github.com/TissueImageAnalytics/tiatoolbox/blob/3e63ea36471dbe1a6088ca317f39107c86a3c914/tiatoolbox/annotation/storage.py#L1319

Fix typo "comparing"

**Review by shaneahmed — APPROVED** [review]

**Pull request merged**